impeller/compiler/switches.cc

```diff
@@ -85,8 +85,15 @@ Switches::Switches(const fml::CommandLine& command_line)
     if (!include_dir_path.data()) {
       continue;
     }
+
+    // fml::OpenDirectoryReadOnly for Windows doesn't handle relative paths
+    // beginning with `../` well, so we build an absolute path.
+    auto include_dir_absolute =
+        ToUtf8(std::filesystem::absolute(std::filesystem::current_path() /
+                                         include_dir_path)
```

Comment:
What happens if `include_dir_path` is already absolute?

Reply:
`include_dir_path` will get cast to a path marked absolute, which will make the append operator [discard](https://en.cppreference.com/w/cpp/filesystem/path/append) the LHS.